lib/command_line_reporter/table.rb

```diff
@@ -62,8 +60,7 @@ def auto_adjust_widths
     def separator(type = 'middle')
       left, center, right, bar = use_utf8? ? utf8_separator(type) : ascii_separator
 
-      separator_str = left + self.rows[0].columns.map {|c| bar * (c.width + 2)}.join(center) + right
-      separator_str.send(self.border_color) if self.border_color
+      left + self.rows[0].columns.map {|c| bar * (c.width + 2)}.join(center) + right
```

Comment:
Line is too long. [84/80]<br>Redundant `self` detected.<br>Space between { and | missing.<br>Space missing inside }.